koa-crud/src/use-cases/promos/update-promo.ts

```diff
@@ -0,0 +1,31 @@
+import { UseCase, PromosStore } from '../../types';
+
+const updatePromo = ({
+  promoEntity,
+  promosStore,
+  R,
+}: {
+  promosStore: PromosStore;
+  promoEntity;
+  R;
+}): UseCase<boolean> => {
+  return async function ({ id, info }) {
+    const promoExists = await promosStore.promoExistsByFilter({
+      _id: id,
+    });
+
+    if (!promoExists) {
+      throw new Error(`Promo ID doesn't exist`);
+    }
+
+    const promo = await promoEntity(info);
+
+    const truthlyPromo = await R.filter(R.complement(R.isNil))(promo);
```

Comment:
you can refactor this to: 
```ts
R.filter(Boolean)(promo)
```

Reply:
@lafbelmonte  Do we still need to `await` here?

Reply:
No sir, will remove the await.